packages/backstopjs-config/README.md

```diff
@@ -11,12 +11,18 @@ This npm package provides configuration for visual regression, or screenshot, te
 
 ## Development Setup
 
-First install the package and its dependencies. This includes headless Chrome, so will take a moment.
+1. First install the package and its dependencies. This includes headless Chrome, so will take a moment.
 
-1. Install the package. Run this command from the same location as package.json.
-	```language:bash
-	npm install @penskemediacorp/backstopjs-config backstopjs --save-dev
-	```
+	- Install @penskemediacorp/backstopjs-config in the project's assets folder.
+		```language:bash
+		npm install @penskemediacorp/backstopjs-config --save-dev
+		```
+
+	- Pull docker image
+
+		```language:bash
+		docker pull backstopjs/backstopjs
```

Comment:
Can this be a postinstall npm script? Or another way to make sure everything is set up with one step - just `npm install`

Reply:
I personally wouldn't since docker is outside the NPM ecosystem. It feels like an overreach to automatically pull things down / install things that are outside of the NPM ecosystem. I don't know if this is a generally followed rule, just my gut feeling on it.

Reply:
Hmm it's a good point but this addition means that Docker is a dependency of this package so it needs to happen. Do we want to support the use of this package without Docker?

Reply:
> Docker is a dependency of this package so it needs to happen

I disagree here -- not all dependencies have to be automatically installed. This would be mean we'd be responsible for writing a cross-platform script that automatically installs Docker on their system as well.

Take a global NPM package for example -- even being in the same ecosystem, I wouldn't auto-install a global package via postinstall -- it's too invasive, goes outside the scope of the project, and could even impact other projects on the user's computer. It's expected that I will only have global packages installed by me.

I think it's the same for docker images -- should only happen if I pull them, or do something where I expect to be pulling them via the Docker ecosystem (such as running a Dockerfile or docker-compose.yml file).

Reply:
It's a good point, and both of our points are correct. The ultimate answer is that we need this to be automated and to run in the pipeline. 

In the meantime, can we update the language in the docs with something like: "To run the tests outside Docker, you can install the  BackstopJS globally with `npm install -g backstopjs`." And replace the `backstop` script below to simply `backstop`.

Reply:
> answer is that we need this to be automated and to run in the pipeline.

Oh, pipeline gets treated differently - probably involves a shell script or something. I didn't setup pipeline as part of this PR but yes it would be automatic for the pipeline :+1:.

> In the meantime, can we update the language in the docs with something like: "To run the tests outside Docker, you can install the BackstopJS globally with npm install -g backstopjs." And replace the backstop script below to simply backstop.

This unfortunately cannot be optional due to cross-platform rendering discrepancies. Saving references from OSX locally will break all tests on locally run Linux tests or tests run properly via the docker image, and vice-versa. Once pipeline is setup, OSX users would also be unable to get their pipeline to pass if they try to run locally.